### DISCUSSION TITLE
Installing brew on MacOS GH Actions runner

### DISCUSSION THREAD

**Discussion by filipsworks** [Homebrew/discussions#4490] [Getting started]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

Installation script is not prepared for using with `yes` nor `echo` for providing sudo password and is protected from running directly through `sudo` - this make the whole thing ending up in limbo where there seems to be no known way of installing the homebrew on MacOS GH Actions runner through the action itself.

Any ideas of how to hack that w/o making whole own install script?

**Comment by SMillerDev** [comment]
https://github.com/Homebrew/actions/tree/master/setup-homebrew

**Reply by filipsworks** [reply]
Thanks for the reply but either usage instructions are not clear or it's simply not working on MacOS as it checks if brew is already installed and performs ` if ! which brew &>/dev/null; then
        echo "Could not find 'brew' command in PATH or standard locations."
        exit 1
    fi`

**Comment by marcIhm** [comment]
I would be interested too in running my own github runner for testing a homebrew formula (unfortunately I am not admittedt to homebrew-core, so I need to run my own).
But as far as I can see filipsworks is right and the cited runner does not seem to set up homebrew (contradicting its name).
Am I missing something ? Surely I do not have the big picture, so any explanation or hint would be useful too ...
